hpOneView/common.py

```diff
@@ -208,6 +208,7 @@ def wrapper(*args, **kwargs):
     # Uncategorized
     # ------------------------------------
     'unmanaged-devices': '/rest/unmanaged-devices',
+    'vc-migration-manager': '/rest/migratable-vc-domains'
```

Comment:
Looks like this line is unnecessary, since the URI is being defined on the scope of the class VcMigrationManager.